reference/content-types/category_selection.rst

```diff
@@ -11,7 +11,24 @@ section of Sulu. The selection will be saved as an array.
 Parameters
 ----------
 
-No parameters available
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Type
+      - Description
+    * - item_disabled_condition
+      - string
+      - Allows to set a jexl expression that evaluates if an item should be displayed as disabled.
```

Comment:
I think I would link to the `jexl` github page here. Make it easier for people to understand what they can put here :slightly_smiling_face: